packages/combo-box/test/external-filtering.test.js

```diff
@@ -94,18 +85,12 @@ describe('external filtering', () => {
       expect(comboBox.hasAttribute('loading')).to.be.false;
       expect(overlay.hasAttribute('loading')).to.be.false;
     });
-
-    it('should not perform measurements when loading changes if not opened', () => {
```

Comment:
This test was added in https://github.com/vaadin/vaadin-combo-box/pull/700 which was a performance optimization. The corresponding `_setOverlayHeight` method has been removed in https://github.com/vaadin/web-components/pull/2497 so the test is now pointless.